include/splitvector/splitvec.h

```diff
@@ -84,14 +86,22 @@ template <typename T, class Allocator = DefaultAllocator<T>>
 class SplitVector {
 
 private:
+   Allocator _allocator;         // Allocator used to allocate and deallocate memory;
    T* _data = nullptr;           // actual pointer to our data
-   size_t* _size;                // number of elements in vector.
-   size_t* _capacity;            // number of allocated elements
+   SplitInfo* _info;             // store size and capacity 
```

Comment:
There are still comments referencing `_capacity` instead of `_info->capacity` and  `_size` instead of `_info->size`